pkg/oracles/contracts/LPOracleFactoryBase.sol

```diff
@@ -25,6 +25,7 @@ abstract contract LPOracleFactoryBase is ILPOracleFactoryBase, ISequencerUptimeF
     // accurately reflect the state. These values are passed to the oracle contracts on creation.
     AggregatorV3Interface internal immutable _sequencerUptimeFeed;
     uint256 internal immutable _uptimeResyncWindow;
+    bool internal immutable _shouldUseBlockTimeForOldestFeedUpdate;
```

Comment:
Hm.

I don't think we'll want two different factories just for this.
I'd suggest adding this as a parameter to the `create` function. The flag should also be part of the oracle ID, as you can potentially create two oracles with the same feeds and different flag settings. This way the oracle is unequivocally identified by its setup.